modules/UI/videolayout/RemoteVideo.js

```diff
@@ -245,6 +245,10 @@ RemoteVideo.prototype._requestRemoteControlPermissions = function () {
             {user: this.user.getDisplayName()
                 || interfaceConfig.DEFAULT_REMOTE_DISPLAY_NAME}
         );
+        let pinnedId = this.VideoLayout.getPinnedId();
```

Comment:
Pinning the controlled participant when the reply from the request for permissions is received. Actually even if the controlled participant has denied the request, we will pin him again. Do you think we should pin the participant only if the permissions have been granted?